addons/medical_treatment/initSettings.sqf

```diff
@@ -181,6 +181,15 @@
     true
 ] call CBA_settings_fnc_init;
 
+[
```

Comment:
Trivial, but, move above the `allowSelfIV` setting to match ordering of other treatment type settings.